openapi/endpoints/funding-channel/funding-channels-list.yaml

```diff
@@ -10,14 +10,25 @@ get:
     required: true
     schema:
       type: string
-
+  - name: limit
+    in: query
+    description: amount of records to return
```

Comment:
i think its max 100? you can double check in source code and add the max in here like with list funding source interactions